docs: Document using actionset labels to extend logs (#2981)
kanisterio/kanister: docs/tasks/logs_labels.rst

> Configuring logs for specific ActionSets
> ----------------------------

Reviewer: ```suggestion
Configuring logs for specific ActionSets
----------------------------------------
```

Kanister uses structured logging to ensure that its logs can be easily
categorized, indexed and searched by downstream log aggregation software.

> Extra fields can be added to log records for specific ActionSet.
> To do that the ActionSet needs to have a label with the ``kanister.io/`` prefix.

Reviewer: ```suggestion
Extra fields can be added to the logs related to a specific ActionSet by adding a label in the ActionSet with ``kanister.io`` prefix.
```
